docs/relations_guide.md

```diff
@@ -1,123 +1,255 @@
+
+-   *subPropertyOf*: overlaps \# if X has\_soma\_location some Y, then X overlaps some Y)
+
+-   *domain*: neuron \# X has\_soma\_location some Y => X is inferred to be a subClassOf neuron
+
+-   *property chain*: has\_soma\_location o part\_of --> has\_soma\_location \# If x has soma location y and y is part\_of z, then x has\_soma\_location\_z
+
+**Example of reasoning with the property chain:**
+
+'cortical interneuron' equivalentTo 'interneuron' that has\_soma\_location some 'cerebral cortex' 'rosehip neuron'
+
+'rosehip neuron' subClassOf interneuron  and has\_soma\_location some 'cortical layer 1' 
+
+'cortical layer 1'  subClassOf part\_of some 'cerebral cortex
+
+=> 'rosehip neuron' subClassOf 'cortical interneuron'
+
+### sends synaptic output to region
+
+A relationship between a neuron and a region, where the neuron has a functionally relevant number of output synapses in that region.
+
+'[<u>adult basket subesophageal neuron</u>](http://purl.obolibrary.org/obo/FBbt_00051856)' SubClassOf [<u>sends synaptic output to region</u>](https://www.ebi.ac.uk/ols4/ontologies/fbbt/properties/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FRO_0013003?lang=en)
+*some* [<u>inferior posterior slope</u>](https://www.ebi.ac.uk/ols4/ontologies/fbbt/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FFBbt_00045046?lang=en)
 
-SubClassOf(obo:CL_2000048 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0002257))
+### receives synaptic input in region
 
-There is also a dedicated set of relations for recording the location of synaptic terminals and projections of neurons.  See Recording synaptic connectivity below.
+A relationship between a neuron and a region, where the neuron has a functionally relevant number of output synapses:
 
-### Recording anatomical location (cells in immaterial spaces)
+e.g. '[<u>adult basket subesophageal neuron</u>](http://purl.obolibrary.org/obo/FBbt_00051856)' SubClassOf ‘[<u>receives synaptic input in region</u>](https://www.ebi.ac.uk/ols4/ontologies/fbbt/properties/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FRO_0013002?lang=en)*’
+some ‘*<u>[superior posterior slope](https://www.ebi.ac.uk/ols4/ontologies/fbbt/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FFBbt_00045040?lang=en)’</u>
 
-To record the location of a cell in an anatomical space (e.g., a sinus), [**'located in'**](http://purl.obolibrary.org/obo/RO_0001025) is used. For example, [lymph node marginal reticular cell](http://purl.obolibrary.org/obo/CL_0009103) has the following subclass:
+### fasciculates\_with
 
+Use this to record the tracts or nerves that a neuron’s projections fasciculate with.
 
-[**'located in'**](http://purl.obolibrary.org/obo/RO_0001025) *some* ['subcapsular sinus of lymph node'](http://purl.obolibrary.org/obo/UBERON_0005463)
+e.g. ‘Betz cell’ subClasssOf ‘fasciculates with’ some ‘corticospinal tract’.
 
+subPropertyOf: overlaps
 
-### Recording function
+domain: neuron
+range: neuron projection bundle
```

Comment:
Purely a formatting issue: needs a blank line here, otherwise this will be displayed on a single line (`domain: neuron range: neuron projection bundle`).

Reply:
Fixed in [5cd8021](https://github.com/obophenotype/cell-ontology/commit/5cd80216a7dd13372a649354ab38bc56fcc0d97a)